benchmarks/distributed/rpc/ps/README.md

```diff
@@ -0,0 +1,62 @@
+# RPC PS Benchmark
```

Comment:
nit: Change folder name from ps -> parameter_server? `.../rpc/parameter_server` sounds more intuitive than `.../rpc/ps` to me

Reply:
okay.